.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: onetest
+
+on: [push]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  one_test:
+    runs-on: ubuntu-latest
+    permissions:
+     contents: read
+     packages: write
+
+    steps:
+    - name: Git clone my repo
+      uses: actions/checkout@v3
+
+    - name: Clone repo
+      run: |
+        git clone https://github.com/ganatan/angular-react-starter.git  
+        
+    - name: START angular test
+      run: |
+        cd angular-react-starter
+        cd angular
+        npm install
+  
+  
+    - name: Build
+      run: |
+        cd angular-react-starter
+        cd angular
+        npm run ng build
+        cd dist     
+        
+    - name: Make images
+      run: |
+        docker build -t myang -f /home/runner/work/angular-react-starter/angular-react-starter/Dockerfile .
+    
+      
+    - name: Log in to the Container registry
+      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+      with:
+        images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+
+    - name: Build and push Docker image
+      uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+      with:
+        context: .
+        push: true
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
```

Comment:
Please implement the step to calculate the time/duration of your pipeline execution as per the test requirements